### PR TITLE
Calibre 5.32.0

### PR DESCRIPTION
**Pull request by extrowerk**
Reader still not working as
```
from qt.webengine import QWebEngineUrlScheme
ImportError: cannot import name 'QWebEngineUrlScheme' from 'qt.webengine'
```

I assume we need to rebuild PyQt with enabled webengine. 

Otherwise the GUI works.

### CODE REVIEW

**Comment by extrowerk** [comment]
Apparently the WebEngine support resides in a different package: PyQtWebEngine
Lets port that too.

**Comment by korli** [comment]
Still WIP?

File: app-text/calibre/calibre-5.32.0.recipe [hunk 1]
```diff
@@ -175,8 +178,10 @@ BUILD_REQUIRES="
 	devel:libpoppler${secondaryArchSuffix}_qt5
 	devel:libpoppler$secondaryArchSuffix
 	devel:libQt5Core$secondaryArchSuffix
+	devel:libQt5WebEngine$secondaryArchSuffix
 	devel:libQt5WebKit$secondaryArchSuffix
```
Comment (korli) [inline app-text/calibre/calibre-5.32.0.recipe:182]: devel:libQt5WebKit can probably be removed?

Comment (extrowerk) [reply app-text/calibre/calibre-5.32.0.recipe:182]: Yep

**Comment by extrowerk** [comment]
> Still WIP?

No.

**Pull request merged**